app/src/androidTest/java/com/github/houseorganizer/houseorganizer/SampleTest.java

```diff
@@ -0,0 +1,43 @@
+package com.github.houseorganizer.houseorganizer;
+
+
+import static org.junit.Assert.assertEquals;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.ExecutionException;
+
+@RunWith(AndroidJUnit4.class)
+public class SampleTest {
+
+    private static FirebaseFirestore db;
+    private static FirebaseAuth auth;
+
+    @BeforeClass
+    public static void createMockFirebase() throws ExecutionException, InterruptedException {
+        FirebaseTestsHelper.startAuthEmulator();
+        FirebaseTestsHelper.startFirestoreEmulator();
+        FirebaseTestsHelper.setUpFirebase();
+
+        db = FirebaseFirestore.getInstance();
+        auth = FirebaseAuth.getInstance();
+    }
+
+    @AfterClass
+    public static void signOut(){
+        auth.signOut();
+    }
+
+    @Test
+    public void sampleTest() {
+        assertEquals("user_1@test.com", auth.getCurrentUser().getEmail().toString());
```

Comment:
i guess toString is not needed since getEmail returns a String

Reply:
As this is a sample test I will not change it but you are right, we should just write getEmail in other tests 👍